types/markdown-it/lib/ruler.d.mts

```diff
@@ -134,20 +127,20 @@ declare class Ruler<T> {
      * Enable rules with given names, and disable everything else. If any rule name
      * not found - throw Error. Errors can be disabled by second param.
      *
-     * See also [[Ruler.disable]], [[Ruler.enable]].
+     * @see {@link Ruler.disable}, {@link Ruler.enable}
      *
      * @param list list of rule names to enable (whitelist).
      * @param ignoreInvalid set `true` to ignore errors when rule not found.
      */
-    enableOnly(list: string | string[], ignoreInvalid?: boolean): string[];
```

Comment:
this type was wrong. enableOnly doesn't return anything (see https://unpkg.com/browse/markdown-it@14.1.0/lib/ruler.mjs line 281 to 286)